CMakeLists.txt

```diff
@@ -279,12 +279,11 @@ IF(CMAKE_HOST_UNIX AND NOT FORCE_UNSUPPORTED_COMPILER
       IF(LINUX_RHEL6)
         SET(ALTERNATIVE_PATHS "/opt/rh/devtoolset-8")
       ELSEIF(LINUX_RHEL7)
-        # ARM requires GCC 10, but we still build releases for
-        # x86_64 on GCC 8.
+        # ARM requires GCC 10. GCC 10 is faster than GCC 9
         IF(MY_HOST_MACHINE_NAME MATCHES "aarch64")
           SET(ALTERNATIVE_PATHS "/opt/rh/devtoolset-10")
         ELSE()
-          SET(ALTERNATIVE_PATHS "/opt/rh/devtoolset-9")
+          SET(ALTERNATIVE_PATHS "/opt/rh/devtoolset-10")
```

Comment:
Do you also want to adjust the `update-alternatives` command in Dockerfile.ubuntu22?

Reply:
ok, done